src/spec/integration/export.spec.coffee

```diff
@@ -205,3 +206,36 @@ describe 'Export integration tests', ->
     .catch (err) -> done _.prettify(err)
     .finally ->
       tempDir.removeCallback()
+
+  it 'should export labels of lenum and set of lenum', (done) ->
+    encoding = 'win1250'
+    template =
+    '''
+      productType,name,variantId,attr-lenum-n.en,attr-set-lenum-n.en,žškřďťň
+      '''
+    file = '/tmp/output.csv'
```

Comment:
Can we name this `outputLocation`? Every time I read file, I expect to have a file, not a path to a file 😜